lib/tasks/check_force_delete.rake

```diff
@@ -3,8 +3,7 @@ task check_force_delete: :environment do
   validations = ValidationEvent.failed.where(validation_eventable_type: 'Contact').uniq(&:validation_eventable_id)
 
   invalid_contact_ids = validations.select do |validation|
-    contact = validation.validation_eventable
-    contact.need_to_start_force_delete? || contact.need_to_lift_force_delete?
+    validation.validation_eventable.need_to_start_force_delete?
```

Comment:
where did you hide the check for need_to_lift_force_delete? how will he understand that force delete can be removed on a specific domain?